src/core/ddsi/tests/sysdeps.c

```diff
@@ -162,7 +162,7 @@ static ddsrt_avl_tree_t helper_tree = {
   &helper_node.avlnode
 };
 
-static void log_stacktrace_helper (void *vnode, void *varg)
+static void log_stacktrace_helper (const void *vnode, void *varg)
```

Comment:
If you make this one `const` (I'm in favour of that change) then `log_stacktrace_thread` should use `ddsrt_avl_const_walk`